lib/Qandidate/Toggle/Serializer/ContextSerializer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Qandidate\Toggle\Serializer;
+
+use Qandidate\Toggle\Context;
+
+class ContextSerializer
+{
+    /**
+     * @param array $data
+     *
+     * @return Context
+     */
+    public function deserialize(array $data)
+    {
+        $this->assertHasKey('context', $data);
```

Comment:
Do we need this class? This seems like logic for a service/controller in the API to me.